docs/api/migration-guides/sessions.mdx

```diff
@@ -0,0 +1,94 @@
+
+<span id="two-vqe"></span>
+### Example: Run two VQE algorithms in a session by using threading
+
+```python
+from concurrent.futures import ThreadPoolExecutor
+from qiskit_ibm_runtime import  Session, EstimatorV2 as Estimator
+
+def minimize_thread(estimator, method):
+    minimize(cost_func, x0, args=(ansatz, hamiltonian, estimator), method=method)
+
+with Session(backend=backend), ThreadPoolExecutor() as executor:
+    # Add tags to differentiate jobs from different workloads.
+    estimator1.options.environment.job_tags = "cobyla"
+    estimator1.options.environment.job_tags = "nelder-mead"
+
+    cobyla_result = executor.submit(minimize_thread, estimator1, "cobyla").result()
+    nelder_mead_result = executor.submit(minimize_thread, estimator2, "nelder-mead").result()
+```
+
+### Extended sessions
+
+Before this release, sessions were terminated after the active window, as shown in the following image.
+
+![This image shows four jobs.  Between each is the interactive TTL (time to live).  The active window starts when the first job starts and ends after the last job is completed. After the final job completes, the active window ends and the session terminates.](/images/run/sessions/SessionTTL.svg 'Figure 1: Session behavior prior to extended sessions')
+
+With extended sessions, the session pauses after the active window ends. The session is resumed if more session jobs exit the queue.
+
+![This image shows multiple sets of jobs.  Within each set, and between each job, is the interactive TTL (time to live).  The active window starts when the first job starts and ends after the last job is completed. After the final job of the first set of jobs completes, the active window ends and the session is paused.  Another set of jobs then starts and jobs continue in a similar manner.](/images/run/sessions/ExtendedSession.svg 'Figure 2: Extended sessions')
+
+## Batch
+
+Submit multiple primitive jobs simultaneously. When batching, classical processing is done in parallel. No session jobs can start when batch jobs are being processed.
+
+<span id="divide"></span>
+### Example: divide a 500-circuit job into five 100-circuit jobs and run them in batch
+```python
+from qiskit_ibm_runtime import Batch, SamplerV2 as Sampler
+
+max_circuits = 100
+jobs = []
+start_idx = 0
+
+with Batch(backend):
+    sampler = Sampler()
+    while start_idx < len(circuits):
+        end_idx = start_idx + max_circuits
+        jobs.append(sampler.run([(circuits[start_ids:end_idx],)]))
+        start_idx = end_idx
+```
+
+## Sessions versus batch QPU usage
+
+QPU usage is a measurement of the amount of time the system is locked for your workload.
+
+* Session QPU usage is the entire active session time.
+* Batch QPU usage is the amount of time all jobs spend on the system.
+* Single job QPU usage is the quantum time they use in processing.
```

Comment:
it seems strange to call the Session usage QPU usage since it's not QPU time - maybe this is unique to how qiskit exposes the field to users

Reply:
This might be left over from old QPU usage discussions.  @jyu00, can you chime in here?

Reply:
Looks like the UI calls it "Qiskit runtime usage", so I can just use that.